iree/base/CMakeLists.txt

```diff
@@ -215,6 +216,8 @@ iree_cc_library(
     "internal/init_internal.cc"
   DEPS
     absl::flags
+    absl::flags_parse
```

Comment:
@ScottTodd Having fixed this locally, I noticed that the bazel BUILD file only has a dep on "@com_google_absl//absl/flags:parse". Therefore, I just replaced `absl::flags` by `absl::flags_parse`, which at least is sufficient for a successful compilation. Is there a specific reason to still rely on `absl::flags`?

Reply:
No, there is no specific reason. That dep could be removed. Extra dependencies usually aren't a problem, though they can increase build times.

We're also considering automating parts of the build system maintenance using https://github.com/google/bazel-to-cmake or a custom set of scripts. That, combined with tools like those in https://github.com/bazelbuild/buildtools , will help us keep dependencies minimal and synchronized between build systems.